docker/prod/nodejs/nginx_staging.conf

```diff
@@ -2,6 +2,14 @@ upstream django_app {
   server django:8000 fail_timeout=0;
 }
 
+upstream node_exporter {
+  server nodeexporter:9100 fail_timeout=0;
+}
+
+upstream prometheus_pushgateway {
+  server pushgateway:9091 fail_timeout=0;
```

Comment:
We need only the node exporter on the same instance as EvalAI right?

Reply:
@Ram81 yes